catalog/plugins/custom_saves_folders/plugin.py

```
@@ -1550,6 +1552,59 @@ def _apply_name_to_targets(self, game_id: str, folder_name: str, task_runtime=No
+                logger.debug("Skipping unreadable mod config: %s", config_path)
+                continue
+            if str(config_data.get("game", "") or "").strip() != game_id:
+                continue
+            mod_id = _canonical_mod_id(get_mod_id(config_data) or folder)
+            if not mod_id or mod_id in seen_ids:
+                continue
+            seen_ids.add(mod_id)
+            if isinstance(context, ShortcutPluginContext):
+                for step in context.shortcut_config["launch_plan"]["patch_plan"]["sections"][game_id]:
+                    for used_mod in step:
+                        if mod_id == used_mod: used_mods.append(folder)
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)